.circleci/config.yml

```diff
@@ -187,29 +187,28 @@ commands:
                     - yarn-packages-{{ arch }}-{{ checksum "yarn.lock" }}
                 name: Restore Yarn Package Cache
             - run:
-                command: yarn install --frozen-lockfile --cache-folder ~/.cache/yarn
+                command: yarn install --immutable
                 name: Install Dependencies
             - save_cache:
                 key: yarn-packages-{{ arch }}-{{ checksum "yarn.lock" }}
                 name: Save Yarn Package Cache
                 paths:
-                    - ~/.cache/yarn
+                    - .yarn/cache
     restore_yarn_tools_cache:
-        description: Restores and saves the node_modules directories of the tools package.
+        description: Restores and saves the tools node_modules directory
         steps:
             - restore_cache:
                 key: 2-yarn-{{ arch }}-{{ checksum "tools/yarn.lock" }}-{{ checksum "tools/package.json" }}
                 name: Restore Yarn Tools Package Cache
             - run:
-                command: yarn --frozen-lockfile --cache-folder ~/.cache/yarn-tools
-                name: Yarn tools
+                command: yarn install --immutable
+                name: Install Dependencies
                 working_directory: tools
             - save_cache:
                 key: 2-yarn-{{ arch }}-{{ checksum "tools/yarn.lock" }}-{{ checksum "tools/package.json" }}
                 name: Save Yarn Tools Package Cache
                 paths:
-                    - ~/.cache/yarn-tools
-                    - tools/node_modules
+                    - tools/.yarn/cache
```

Comment:
Have you checked, if caching is working?

Reply:
yesss 👍🏼